db/seeds.rb

```diff
@@ -1,46 +1,227 @@
+seed_event.rooms.create(name: "Moon Room", room_number: "MOON", level: "6", address: "123 Universe Drive", capacity: 150, event_id: seed_event.id)
+seed_event.rooms.create(name: "Venus Theater", room_number: "VEN-T", level: "2", address: "123 Universe Drive", capacity: 75, event_id: seed_event.id)
 
 # Event Team
 seed_event.teammates.create(user: organizer, email: organizer.email, role: "organizer", state: Teammate::ACCEPTED, notifications: false)
-seed_event.teammates.create(user: track_director, email: track_director.email, role: "organizer", state: Teammate::ACCEPTED) # < update to program team
+seed_event.teammates.create(user: track_director, email: track_director.email, role: "program team", state: Teammate::ACCEPTED)
 seed_event.teammates.create(user: reviewer, email: reviewer.email, role: "reviewer", state: Teammate::ACCEPTED)
 seed_event.teammates.create(user: speaker_reviewer, email: speaker_reviewer.email, role: "reviewer", state: Teammate::ACCEPTED)
 
+# Proposals - there are no proposals that are either fully "accepted" or offically "not accepted"
+submitted_proposal_1 = seed_event.proposals.create(event_id: seed_event.id,
+                                                   uuid: "abc123",
+                                                   title: "Honey Bees",
+                                                   abstract: "We will discuss the vital importance of pollinators and how we can help them thrive.",
+                                                   details: "Why we need pollinators, what plants they love most, basics of how to start your own hive.",
+                                                   pitch: "Learning to be stewards for our insect friends is essential to keeping some of our favorite foods around!",
+                                                   session_format_id: long_session.id,
+                                                   track_id: track_1.id)
+
+submitted_proposal_2 = seed_event.proposals.create(event_id: seed_event.id,
+                                                   uuid: "def456",
+                                                   title: "Coffee Talk",
+                                                   abstract: "We go over what makes a great cup of coffee as well as different methods of preparation.",
+                                                   details: "We will talk about the coffee plant itself, the roasting process, and prepare the same beans in a variety of different ways to compare the flavor and nuances.",
+                                                   pitch: "You need coffee to live happily, why not be drinking the best tasting versions of your favorite drug?",
+                                                   session_format_id: long_session.id,
+                                                   track_id: track_2.id)
+
+waitlisted_proposal = seed_event.proposals.create(event_id: seed_event.id,
```

Comment:
I would make this soft_waitlisted vs waitlisted for now since waitlisting is a final state.